src/applications/vaos/utils/data.js

```diff
@@ -196,6 +196,7 @@ export function transformFormToAppointment(state) {
   );
 
   return {
+    appointmentType: getTypeOfCare(data).name,
```

Comment:
@saneshark and I are not sure if this is necessary, but passing it along just in case